app/jobs/daily_broadcast_auctions_job.rb

```diff
@@ -0,0 +1,16 @@
+class DailyBroadcastAuctionsJob < ApplicationJob
+  def perform
+    User.where(auction_newsletter: true).each do |subscriber|
+      NotificationMailer.daily_auctions_broadcast_email(subscriber, auctions_today).deliver_later
+    end
+  end
+
+  def auctions_today
+    timeframe = Time.zone.today.beginning_of_day..Time.zone.today.end_of_day
+    Auction.where(starts_at: timeframe).to_ary
```

Comment:
Why `.to_ary` is needed?

Reply:
Not sure what I'm missing, but without `.to_ary` I receive `ActiveJob::SerializationError ` when creating domain list in mail view. Tips?

Reply:
I suppose its totally OK. The result of #auctions_today method is used in `NotificationMailer.daily_auctions_broadcast_email(subscriber, auctions_today).deliver_later`
The point is in `deliver_later`, which is processed by ActiveJob.
ActiveJob currently supports String, Integer, Float, NilClass, TrueClass, FalseClass, BigDecimal, Symbol, Date, Time, DateTime, ActiveSupport::TimeWithZone, ActiveSupport::Duration, Hash, ActiveSupport::HashWithIndifferentAccess, Array or GlobalID::Identification instances (https://api.rubyonrails.org/classes/ActiveJob/SerializationError.html) , so here we can:
- either pass an array of something and find object in mailer;
- or use custom serializer which can serialize ActiveRecord_Relation.